### PR TITLE
Add appdata file

### PR DESCRIPTION
**Pull request by mattiaverga**
Add appdata file in linux installations.

See https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps

### CODE REVIEW

File: phd2.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <p>
+        PHD2 is telescope guiding software that simplifies the process
+        of tracking a guide star, letting you concentrate on other aspects of
+        deep-sky imaging or spectroscopy.
+        </p>
+        <ul>
+            <li>Easy-to-use, “push here dummy” guiding for beginners</li>
+            <li>Sophisticated guiding and analysis tools for experienced users</li>
+            <li>Extensive support for commonly-used equipment</li>
+            <li>Easy upgrade/co-installation with PHD1, including many new features</li>
```
Comment (d33psky) [inline phd2.appdata.xml]: This line makes sense for Windows (and maybe Mac) but no sense for Linux

File: phd2.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Mattia Verga <mattia.verga@tiscali.it> -->
+<component type="desktop-application">
+    <id>phd2.desktop</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>BSD-3-Clause</project_license>
```
Comment (d33psky) [inline phd2.appdata.xml]: Is this the license we want ?

File: phd2.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <name>Comical</name>
```
Comment (d33psky) [inline phd2.appdata.xml]: What/who is 'comical' ?

File: phd2.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    <launchable type="desktop-id">phd2.desktop</launchable>
+    
+    <screenshots>
+        <screenshot type="default">
+            <caption>The main window</caption>
+            <image>http://openphdguiding.org/wp-content/uploads/2015/06/capture_21.jpg</image>
+        </screenshot>
+        <screenshot>
+            <caption>The log viewer</caption>
+            <image>http://openphdguiding.org/wp-content/uploads/2015/05/phdlogviewer_sm.png</image>
```
Comment (d33psky) [inline phd2.appdata.xml]: Do we have the logviewer for Mac and Linux ?

File: phd2.appdata.xml (hunk 2 above)
Comment (agalasso) [inline phd2.appdata.xml]: personally I think we should switch to GPL, but that will require approval from all the contributors who have put their names on license text throughout the code

File: phd2.appdata.xml (hunk 4 above)
Comment (agalasso) [inline phd2.appdata.xml]: yes: http://adgsoftware.com/phd2utils/ but we only have Patrick's PPA Ubuntu pre-built .deb, other distros need to build from source

File: phd2.appdata.xml (hunk 2 above)
Comment (mattiaverga) [inline phd2.appdata.xml]: The LICENSE.txt file distributed in sources is a BSD-3-Clause, so I put that in <project_license>.

The <metadata_license> only refers to the appdata file itself.

File: phd2.appdata.xml (hunk 3 above)
Comment (mattiaverga) [inline phd2.appdata.xml]: My fault... a copy/paste error 😮 Should be PHD2.

File: phd2.appdata.xml (hunk 4 above)
Comment (mattiaverga) [inline phd2.appdata.xml]: Ah, I thought it was included in the main PHD2 program... I will remove this screenshot.

**Pull request merged**